cal/cal/static/templates/categoryDetails.html

```diff
@@ -1,8 +1,11 @@
 <div>
-  <h3 class="well">Average hours per week: {{ categoryHours }}<h3>
-  <nvd3 options="categoryLine" data="categoryDetails"></nvd3>
+  <h3 class="well">Average hours per {{$ctrl.timeStep}}: {{ $ctrl.averageHours }}<h3>
+  <button ng-click='$ctrl.showDaily()'>Daily</buton>
+  <button ng-click='$ctrl.showWeekly()'>Weekly</button>
+  <button ng-click='$ctrl.showMonthly()'>Monthly</button>
+  <nvd3 options="$ctrl.categoryLine" data="$ctrl.ctrlDetails"></nvd3>
```

Comment:
I made a remark earlier, but making a review comment to be concrete: I think `ctrlDetails` is not a very clear name. @jalagar any suggestions?